tests/Makefile.tests_common

```diff
@@ -1,7 +1,7 @@
 APPLICATION ?= tests_$(notdir $(patsubst %/,%,$(CURDIR)))
 
 # set and redirect default paths
-TESTBASE ?= $(CURDIR)/../../
+TESTBASE ?= $(CURDIR)/../..
```

Comment:
does it make difference? or is specifically import?

Reply:
Nope, I remember it just looked a bit silly on a printout having two slashes.